modules/chapter1/pages/section1.adoc

```diff
@@ -251,33 +251,39 @@ CMD ["app:app"]
 
 . Build and push the image to an image registry
 +
-```shell
-podman login quay.io
-podman build -t purchase-predictor:1.0 .
-podman tag purchase-predictor:1.0 quay.io/user_name/purchase-predictor:1.0
-podman push quay.io/user_name/purchase-predictor:1.0
-```
+
+[source,console]
+----
+$ podman login quay.io
+$ podman build -t purchase-predictor:1.0 .
+$ podman tag purchase-predictor:1.0 quay.io/user_name/purchase-predictor:1.0
```

Comment:
Not a blocker comment, but why creating 2 tags for the same container image?
Why not using the FQN tag in the build step?

Reply:
Maybe for clarity? We would have to ask the original writer of the exercise.